site/source/docs/getting_started/FAQ.rst

```diff
@@ -274,6 +274,12 @@ Another option is to use the ``MODULARIZE`` option, using ``-s MODULARIZE=1``. T
 
 Note that in ``MODULARIZE`` mode we do not look for a global Module object for default values. Default values must be passed as a parameter to the factory function.  (see details in settings.js)
 
+Why is Module['onRuntimeInitialized'] not called when my JavaScript file is loaded?
+===================================================================================
+
+When you are compiling your application to a ``.js`` file, you normally use ``emcc`` or ``em++`` flag ``--post-js`` to load the file. Then, your next ``<script>`` tag will likely have a ``src`` attribute pointing to your client or appliciation code, such as ``index.js``, ``main.js``, or ``app.js``.
```

Comment:
I don't quite understand what "use post-js to load the file" means?

Overall maybe it's better to phrase this as: "Asynchronous loading on the Web can be unpredictable. If you put the Emscripten output in a script tag, and another script tag sets `.onRuntimeInitialized` (or `.then()`) to be notified of when it is ready, there is a possibility that the first tag is loaded in its entirety before the second. To avoid that, ..."

And maybe a small code sample to make things more concrete could help?

Reply:
Agreed - this text is too ambiguous, I am connecting two concepts together as one that really should be two separate phases. The `post-js` and `extern-post-js` flag really just concatenates files, it doesn't load them. I will update this